controllers/game_controller.rb

```diff
@@ -0,0 +1,66 @@
+class GameController < BaseController
+  def get
+    render
+  end
+
+  def post
+    proceed_user_params
+    return proceed_finished_game if game.win? || game.lose?
+
+    render
+  end
+
+  private
+
+  def proceed_user_params
+    if @request.params.key?('hint')
+      give_hint
+    elsif @request.params.key?('guess')
+      make_guess
+    end
+  end
+
+  def proceed_lose
+    Rack::Response.new do |response|
+      response.redirect(Constants::LOSE_GAME_PATH)
+    end
```

Comment:
In BaseController you can make redirect method and use it like `redirect(Constants::LOSE_GAME_PATH)

```
def redirect(path)
  Rack::Response.new { redirect( path ) } 
end
```

Reply:
+